src/components/dropdown/dropdown-group.ts

```diff
@@ -26,7 +26,6 @@ export default class IgcDropdownGroupComponent extends LitElement {
   @queryAssignedElements({ flatten: true, selector: 'igc-dropdown-item' })
   public items!: Array<IgcDropdownItemComponent>;
 
-  /** @private */
```

Comment:
Maybe as an alternative make the group items get the size from their parent instead of exposing an additional attribute.
Even with the **private** tag this would show in auto-complete in editors. Not to mention that the dropdown overwrites the value anyway...

Reply:
Agreed, the way you refactored it is likely better, however.. when the child looks for its parent explicitly it won't be super friendly for extended components (looking at the select, combo maybe). Perhaps generalize the `parent` with some base interface and move the query for it in a `protected getParent()` or something similar.